molecule/default/tests/test_default.py

```diff
@@ -56,15 +46,33 @@ def test_redhat_updated_time(host):
         assert (datetime.datetime.now() - last_update).total_seconds() <= 20 * 60
 
 
-def test_redhat_updated_command_output(host):
-    """Test that RedHat instances were updated."""
-    if (
-        host.system_info.distribution == "amzn"
-        or host.system_info.distribution == "fedora"
-    ):
+def test_yum_updated_command_output(host):
+    """Test that yum-based instances were updated."""
+    if host.system_info.distribution in ["amzn"]:
         yum_output = host.run("yum update")
         # If the update succeeded or there was nothing to update
```

Comment:
I believe this should be updated with the change to the assertion.

Reply:
Please see commit d2c0824.